frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -5,6 +5,8 @@ This module contains miscellaneous helper functions for FileManager
 local Device = require("device")
 local DocSettings = require("docsettings")
 local util = require("ffi/util")
+local _ = require("gettext")
+local util = require("util")
```

Comment:
Pssht this should break horribly.

Reply:
:( ?

Reply:
local *util* = require("ffi/util")
local _ = require("gettext")
local util = require("util")

Reply:
Oh, I am an idiot, will fix that lol